### ISSUE TITLE
Need database for lessons

### ISSUE THREAD

**Issue by rozig** [YunOSC/PyFun-Backend#34]
@jackey8616 I think its time to add database for lessons. Writing lesson data in the code is not the right way to do it. At least, if you don't want to create a database lets save lesson data in JSON. What do you think?

**Comment by Sean2525** [comment]
🤔 I think JSON is a good choice.

**Comment by rozig** [comment]
I'm wondering why are you avoiding database ..

**Comment by Sean2525** [comment]
@rozig Because it will be difficult to maintain and contribute.

**Comment by jackey8616** [comment]
@rozig
Your idea is very very good. And indeed, it's time to consider about centralized each lessons code with storage, such as JSON or DB that you mention earlier.
This platform, we strongly hope can running without any storage service, because we are not trying to running as a website to host it.
BUT, here is a BIG BUT,
For a complete service, I think a DB design is necessary for anyone who want to host it.

**Comment by jackey8616** [comment]
In my opinion is, we can make a simple toggle to switch b/w storage or not.
And I recommend using MongoDB to reduce the complexity of designing a schema.

**Comment by rozig** [comment]
Yes, we can use MongoDB. Can you add MongoDB integration?

**Comment by sawani02** [comment]
Yes, mongodb is better choice, I can help with integration. Please do let me know!

**Comment by jackey8616** [comment]
@sawani02 
If you have any good way at implement MongoDB, just fork and send a PR.
We can review the code and discuss more.

**Comment by sawani02** [comment]
@jackey8616 , Sure! I will implement it and will let you know, I've not yet push the code on github
